### ISSUE TITLE
Long text break message and have extra margins

### ISSUE THREAD

**Issue by Ganzolo** [MessageKit/MessageKit#72]
Hi,

When I try to input very long messages I get an extra margin at the top and the bottom of each bubbles.

![simulator screen shot 16 aug 2017 14 32 42](https://user-images.githubusercontent.com/13142882/29363309-c57f2f0e-828f-11e7-9267-42690c7e22cc.png)

After investigating I realised that if I change in MessagesCollectionViewFlowLayout.Swift (line 64) :

`messageLabelFont = UIFont.preferredFont(forTextStyle: .body)`

by 

`messageLabelFont = UIFont.systemFont(ofSize: 17) // This is the font used anyway in my UILabel`

![simulator screen shot 16 aug 2017 14 36 21](https://user-images.githubusercontent.com/13142882/29363437-4c69f15c-8290-11e7-9a60-658ffe44c88c.png)

It works. Hope this helps.

Best regards,

**Comment by AndrewLauder** [comment]
Is this a valid issue? If so, @Ganzolo, care to do a PR, or would you like me to do that?

**Comment by SD10** [comment]
@Ganzolo Are you changing the font for the `MessageLabel`? It should use the same font provided in the `MessagesCollectionViewFlowLayout`. 

@AndrewLauder I haven't looked into this so I don't know if it's a bug or not. Looks like something is off though. I hope to review it in a bit.

**Comment by Ganzolo** [comment]
@SD10

I did not and I used the same font as the default one in UILabel. It's a very strange issue but it might be related to AutoLayout + preferredFont method.

Here is a person also having trouble with this :

https://stackoverflow.com/questions/34983275/uitext-view-is-getting-randomly-clipped-when-using-autolayout-and-or-preferredfo

@AndrewLauder 
My quick fix of hard coding default font size is not good. If you confirm the issue, we need to get rid of preferredFont method. I'll let u do PR then.

Best regards
